### PR TITLE
Lazy Loading Images

### PR DESCRIPTION
**Pull request by Dnouv**
This PR adds the support for lazy loading images with fade-in animation effect to improve performance, using [lozad](https://www.npmjs.com/package/lozad), a lightweight lazy loading package. The cover image gets loaded after it crosses 30% of the viewport and 10% of its height. Here is a vid of the feature.

https://user-images.githubusercontent.com/61188295/124345764-6ce12980-dbf8-11eb-9244-3c7058d717a2.mp4

Feedbacks are most welcomed. Thank you!

### CODE REVIEW

**Review by jvoisin — APPROVED** [review]

File: ui/src/album/lazy.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.fade {
```
Comment (deluan) [inline ui/src/album/lazy.css:1]: We should strive to only use Material-UI styles (`useStyles`) and avoid using raw, unscoped CSS

**Review by deluan — CHANGES_REQUESTED** [review]
Sorry for the delay. Looks nice! Just a couple of comments below.

In terms of UX, three issues:
1) IMO, I think it should show the images a bit faster
2) When loading new images (not cached), it briefly shows a white frame around the image: 

https://user-images.githubusercontent.com/331353/126055041-ecb91e27-7a83-4704-9ab8-8f5b7ece0f78.mov

3) When the images are cached, it should show the images even faster. Is there a way to achieve this with this library?

File: ui/src/album/lazy.css (hunk 1 above)
Comment (Dnouv) [reply ui/src/album/lazy.css:1]: Sorry for the late reply @deluan; I will try to replace the unscoped CSS with the `useStyles`; however, the unscoped CSS might be required while implementing custom scrollbars; thoughts?

**Comment by deluan** [comment]
Hey @Dnouv , sorry for the loooong delay. I merged your branch with changes from `master`, fixed conflicts. The only issue I see is that it is too slow to load the images, and the final result is not great. I didn't look into that too much, but if you find a way to speed up the fade a bit, we could merge this.

If you are not available to do this change, please let me know. Thanks anyway for your contribution!

**Comment by Dnouv** [comment]
Hey @deluan, no problem; it is good to see you are back on the project. I may not be able to look into the issue this coming two weeks, but I will make some time for it on weekends (14 Jan). Please feel free to take up the PR if you need it urgently.

Thank you!

**Comment by metalheim** [comment]
Lazy Loading is also now available natively via browser support.
Just giving each `img `tag the property `loading="lazy"` should be enough to let the browser handle the lazy loading on its own without a new dependency.

https://web.dev/browser-level-image-lazy-loading/
https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading#browser_compatibility

**Pull request closed without merging**